src/main/groovy/edu/oregonstate/mist/bluelightphones/db/BlueLightDAO.groovy

```diff
@@ -0,0 +1,37 @@
+package edu.oregonstate.mist.bluelightphones.db
+
+import edu.oregonstate.mist.api.jsonapi.ResourceObject
+import edu.oregonstate.mist.bluelightphones.core.Attributes
+import edu.oregonstate.mist.bluelightphones.core.BlueLightPhoneCoordinates
+import groovy.json.JsonSlurper
+
+class BlueLightDAO {
+
+    private File blueLightFile
+    private JsonSlurper jsonSlurper = new JsonSlurper()
```

Comment:
If this property is only used in one method, does it need to be a class level attribute?